templates/firefox-ios.xliff

```diff
@@ -5994,6 +6018,11 @@ Sent from %2$@ 🦊 %3$@</source>
         <source>Tabs open</source>
         <note>Accessibility label for the tabs button in the toolbar, specifing the number of tabs open.</note>
       </trans-unit>
+      <trans-unit id="Toolbar.Tabs.Button.A11y.LargeContentTitle.v137" xml:space="preserve">
+        <source>Tabs open: %@</source>
+        <note>Large content title for the tabs button in the toolbar, specifying that more than 99 tabs are open.
```

Comment:
@thatswinnie: could you please remove the extra comment here? thanks!

Reply:
@Delphine can you run the export again? I fixed it on main.